g3w-admin/client/views.py

```diff
@@ -216,7 +216,10 @@ def render_to_response(self, context, **response_kwargs):
 
         # Add G3W_CLIENT_COOKIE_SESSION_TOKEN cookie to response
         response = super().render_to_response(context)
-        response.set_cookie(settings.G3W_CLIENT_COOKIE_SESSION_TOKEN, secrets.token_hex(16))
+
+        # Only with https set samesite='None' for cross-site requests, i.e. for cross-site iframe
+        kwargs = {'samesite': 'None', 'secure': True} if self.request.is_secure() else {'samesite': 'Strict'}
+        response.set_cookie(settings.G3W_CLIENT_COOKIE_SESSION_TOKEN, secrets.token_hex(16), **kwargs)
```

Comment:
Based on https://github.com/g3w-suite/g3w-admin/commit/32188805dd1f8e4ee63a74a4eb261c71beab160c and recommended [g3w-admin-portal](https://github.com/g3w-suite/g3w-admin-portal?tab=readme-ov-file#cross-domain-authentication) CORS settings:

```suggestion
if self.request.user.is_anonymous:
            response.set_cookie(
                settings.G3W_CLIENT_COOKIE_SESSION_TOKEN,
                secrets.token_hex(16),
                samesite=settings.SESSION_COOKIE_SAMESITE,
                secure=settings.SESSION_COOKIE_SECURE,
                httponly=settings.SESSION_COOKIE_HTTPONLY,
            )
```